tests_metricflow/snapshots/test_cumulative_metric_rendering.py/SqlQueryPlan/DuckDB/test_cumulative_metric_with_agg_time_and_metric_time__plan0.sql

```diff
@@ -18,7 +18,8 @@ FROM (
     FROM (
       -- Join Self Over Time Range
       SELECT
-        subq_2.metric_time__day AS metric_time__day
+        subq_2.revenue_instance__ds__month AS revenue_instance__ds__month
```

Comment:
I find it hilarious that we were just not including these values. Guess it's not a common query pattern.